web/htdocs/assets/js/codesearch_ui.js

```diff
@@ -49,6 +49,16 @@ var Match = Backbone.Model.extend({
   initialize: function() {
   },
   url: function() {
+    if (this.get('tree') in CodesearchUI.intervalViewRepos) {
+      return this.internalUrl();
+    } else {
+      return this.externalUrl();
+    }
+  },
+  internalUrl: function() {
+    return "/view/" + this.get('tree') + "/" + this.get('path') + "#L" + this.get('lno');
```

Comment:
I think there's a bigger question here around link behaviour--we'll need to figure out some story around when we want to permalink a specific version vs when we just want to link to HEAD. It's easy to change the behaviour later, so I'm most interested right now in making sure the URL format is satisfactory.